test/integration/index-test.js

```diff
@@ -61,7 +64,7 @@ describe(function() {
         to
       })).promise;
 
-      await afterMerge();
+      await afterMerge(tmpPath);
```

Comment:
what does this do?

Reply:
Leftover from an iteration of the testing that didn't make it to the final cut. Reverted